chore(rn): Share integrations documentation with javascript (#7009)
getsentry/sentry-docs: src/platforms/react-native/configuration/integrations/redux.mdx

---
title: Redux Integration
description: "Learn about Sentry's Redux integration."
---

_(Available in version 1.8.0 and above)_

> Redux support is included in the `@sentry/react-native` package since version `1.8.0`. To apply Sentry to Redux, use `Sentry.createReduxEnhancer` at the same place that you initialize your Redux store.

Reviewer: ```suggestion
Redux support is included in the `@sentry/react-native` package, version `1.8.0` and up. To apply Sentry to Redux, use `Sentry.createReduxEnhancer` in the same place you'd initialize in your Redux store.
```
